static/js/auth.jsx

```diff
@@ -9,14 +9,6 @@ var self = module.exports = {
             if (cb) cb(true);
                 return
         }
-        self.getToken(username, pass, (res) => {
```

Comment:
And how does it works now?:) I really even don't remember this part off app. What was the issue?